### PR TITLE
Docs: add "How to manage your installation - Managing profiles"

### LINKED ISSUE #4000: Docs: How to manage profiles

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #4000 

This is a new section that is intended to describe the function of
profiles and the various commands that can be used to work with them.
Note that the explanation on the purpose and details of profiles is kept
to a minimum as this is a How-to guide and is merely intended to help
users with goal oriented problems.

The section quickly explains the various profile commands:

    * verdi profile list
    * verdi profile setdefault
    * verdi profile show
    * verdi profile delete

Information on how further configuration through profiles is reserved
for the next how-to section "Configuring your installation".

### CODE REVIEW

**Comment by sphuber** [comment]
@csadorf I am also working on #4001 which is the section that directly follows the one that I just have written in this PR. I am not sure yet which parts should go in which, so I propose we hold off with the review/merge until I have written the other PR as well and we review them at the same time. We might want to swap the location of certain parts.

File: docs/source/howto/installation.rst [hunk 1]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
 =================
 
-`#4000`_
+Creating profiles
+-----------------
+Each AiiDA installation can have multiple profiles, each of which can have its own individual database and file repository to store the contents of the :ref:`provenance graph<topics:provenance:concepts>`.
+Profiles allow you to run multiple projects completely independently from one another with just a single AiiDA installation and at least one profile is required to run AiiDA.
+A new profile can be created using :ref:`verdi quicksetup<reference:command-line:verdi-quicksetup>` or :ref:`verdi setup<reference:command-line:verdi-setup>`, which works similar to the former but gives more control to the user.
+
+
```
Comment (csadorf) [inline docs/source/howto/installation.rst:19]:
```suggestion
```
extra line

File: docs/source/howto/installation.rst [hunk 2]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
+Listing profiles
+----------------
+The :ref:`verdi profile<reference:command-line:verdi-profile>` command line interface provides various commands to manage the profiles of an AiiDA installation.
+To list the currently configured profiles, you can use ``verdi profile list``:
```
Comment (csadorf) [inline docs/source/howto/installation.rst:23]:
```suggestion
To list the currently configured profiles, use ``verdi profile list``:
```

File: docs/source/howto/installation.rst [hunk 3]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
+.. code:: bash
+
+    Info: configuration folder: /home/user/.virtualenvs/aiida/.aiida
+    * project-one
+      project-two
+
+In this particular example, there are two configured profiles, ``project-one`` and ``project-two``.
+The first one is highlighted and marked with a ``*`` symbol, meaning it is the default profile.
+A profile being the default means simply that any ``verdi`` command will always be executed for that profile.
+Of course you can :ref:`change the profile on a per-call basis<topics:cli:profile>` with the ``--p/--profile`` option.
```
Comment (csadorf) [inline docs/source/howto/installation.rst:34]:
```suggestion
You can :ref:`change the profile on a per-call basis<topics:cli:profile>` with the ``--p/--profile`` option.
```

File: docs/source/howto/installation.rst [hunk 4]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
+If you would like to change the default profile in order to not always have to specify it explicitly, you can use ``verdi profile setdefault PROFILE``.
```
Comment (csadorf) [inline docs/source/howto/installation.rst:35]:
```suggestion
To change the default profile use ``verdi profile setdefault PROFILE``.
```

File: docs/source/howto/installation.rst [hunk 5]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
+
+Showing profiles
+----------------
+Each profile defines various parameters, such as the location of the file repository on the file system and the connection parameters for the database.
+To display these parameters, you can use ``verdi profile show``:
```
Comment (csadorf) [inline docs/source/howto/installation.rst:40]:
```suggestion
To display these parameters, use ``verdi profile show``:
```

File: docs/source/howto/installation.rst [hunk 6]
```diff
@@ -10,7 +10,64 @@ How to manage your installation
...
+Deleting profiles
+-----------------
+A profile can be deleted using the ``verdi profile delete`` command.
+By default, deleting a profile will also delete its file repository and the database.
+This behavior can be changed using the ``--skip-repository`` and ``--skip-db`` options.
+
+.. note::
+
+    In order to delete the database, the system user needs to have the required rights, which is not always guaranteed depending on the system.
+    In such cases, the database deletion may fail and the user will have to perform the deletion manually through PostgreSQL.
```
Comment (csadorf) [inline docs/source/howto/installation.rst:70]:
```suggestion
    The deletion of a database can sometimes fail when the system user has insufficient permissions.
    In that case the database must be manually deleted through PostgreSQL.
```

File: docs/source/index.rst [hunk 7]
```diff
@@ -136,6 +136,7 @@ Welcome to AiiDA's documentation!
 .. toctree::
    :maxdepth: 2
    :caption: Internal architecture
+   :hidden:
```
Comment (csadorf) [inline docs/source/index.rst:139]: This change is unrelated.

Comment (sphuber) [reply docs/source/index.rst:139]: True, but should be done at some point. Want me to remove it from this one?

Comment (csadorf) [reply docs/source/index.rst:139]: It's fine.

Comment (sphuber) [reply docs/source/index.rst:139]: All done
